src/Components/Home/Shop/ShopItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './ShopItem.css';
+
+class ShopItem extends React.Component{
```

Comment:
Convert all classes that either does not have state or won't have state in future, to a stateless functional component.